mnd/mnd/registry/patients/mnd_admin_forms.py

```diff
@@ -181,8 +171,21 @@ def clean_email(self):
     def save(self, commit=True):
         rel = self.cleaned_data.get('relationship')
         rel_info = self.cleaned_data.get('relationship_info')
+        email = self.cleaned_data['email']
+        instance = getattr(self, 'instance', None)
+        carer_instance_update = False
+        if instance and instance.pk:
+            db_instance = PrimaryCarer.objects.get(pk=instance.pk)
```

Comment:
Why does this need to be a separate variable?

Reply:
Because we need to compare against the stored email. Observed that instance.email and email are the same.